examples/codesandbox/angular/package.json

```diff
@@ -18,11 +18,7 @@
     "@angular/platform-browser-dynamic": "^8.0.0",
     "@angular/router": "^8.0.0",
     "@babel/runtime": "^7.8.0",
-    "carbon-components": "~10.34.0",
     "carbon-web-components": "^1.0.0",
```

Comment:
@jeffchew I was thinking... We should update it to this?

```
"carbon-web-components": "^1.19.0",
```

Because before that version, there were no dependencies.